app/controllers/application_controller.rb

```diff
@@ -204,6 +204,8 @@ def require_login
 
     def after_sign_in_path_for(resource)
       if request.referrer =~ /sign_in/
+        return root_path if resource&.is_active_without_security_question_responses?
```

Comment:
Just making sure - does the `root_path` redirect you to security questions page?

Reply:
We don't have a single page for security questions, but the security questions modal appears on the main page